packages/components/src/components/overflow-menu/overflow-menu.js

```diff
@@ -114,6 +115,16 @@ class OverflowMenu extends mixin(
    */
   constructor(element, options) {
     super(element, options);
+
+    // Assuming the static options property is created in mixin but it's unique to Overflow Menu at this time
+    // So I want to keep it scoped to this component. Let me know if there is a different/preferred way to set this property
+    this.options.triggerSelector =
+      this.options.triggerSelector || 'button[aria-haspopup]';
```

Comment:
Thank you for asking @elizabethsjudd - You can set it at [`static get options()`](https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/components/overflow-menu/overflow-menu.js#L327-L343). Such static properties are [propagated to instance properties with the same name](https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/globals/js/mixins/create-component.js#L40-L47).